torch/csrc/jit/frontend/sugared_value.cpp

```diff
@@ -209,6 +209,8 @@ std::shared_ptr<SugaredValue> SimpleValue::attr(
   ErrorReport report(loc);
   report << "'" << value_->type()->repr_str()
          << "' object has no attribute or method '" << field << "'.";
+  report.error_type = ErrorReport::ATTR_NOT_FOUND;
```

Comment:
try to use hasattr so we don't need to change this logic

Reply:
done